src/site/filters/liquid.unit.spec.js

```diff
@@ -0,0 +1,16 @@
+import liquid from 'tinyliquid';
+import { expect } from 'chai';
+
+import registerFilters from './liquid';
+
+registerFilters();
+
+describe('isLaterThan', () => {
+  it('returns true when the left arg is a timestamp later than the right arg', () => {
+    expect(liquid.filters.isLaterThan('2020-01-11', '2016-07-10')).to.be.true;
+  });
+
+  it('returns false when the left arg is a timestamp before the right arg', () => {
+    expect(liquid.filters.isLaterThan('2016-12-11', '2017-01-12')).to.be.false;
+  });
+});
```

Comment:
unit tests for liquid filters is overdue

Reply:
so true! 🔥